vcxagency-node/test/unit/configuration/app-config.spec.js

```diff
@@ -80,114 +88,39 @@ describe('app configuration', () => {
       CERTIFICATE_PATH: path.join(__dirname, 'mock_certs', 'mock_cert.pem'),
       CERTIFICATE_KEY_PATH: path.join(__dirname, 'mock_certs', 'mock_key.pem')
     }
-    validateAppConfig({ ...requiredConfig, ...tlsConfig })
+    await validateAppConfig({ ...getValidEnterpriseAgencyConfig(), ...tlsConfig })
   })
 
-  it('should pass app config validation', async () => {
-    const appConfig = {
-      LOG_LEVEL: 'debug',
-      LOG_ENABLE_INDYSDK: 'false',
-      LOG_JSON_TO_CONSOLE: 'false',
-      SERVER_PORT: '8080',
-      SERVER_MAX_REQUEST_SIZE_KB: '300',
-
-      AGENCY_WALLET_NAME: 'vcxagency-node',
-      AGENCY_DID: 'VsKV7grR1BUE29mG2Fm2kX',
-      AGENCY_SEED_SECRET: '0000000000000000000000000Forward',
-      AGENCY_WALLET_KEY_SECRET: '01234567890123456789',
-
-      REDIS_URL: 'redis://localhost:6379/0',
-
-      PG_STORE_HOST: 'localhost',
-      PG_STORE_PORT: '5432',
-      PG_STORE_ACCOUNT: 'postgres',
-      PG_STORE_PASSWORD_SECRET: 'mysecretpassword',
-      PG_STORE_DATABASE: 'agency-storage',
-
-      PG_WALLET_ACCOUNT: 'postgres',
-      PG_WALLET_PASSWORD_SECRET: 'mysecretpassword',
-      PG_WALLET_ADMIN_ACCOUNT: 'postgres',
-      PG_WALLET_ADMIN_PASSWORD_SECRET: 'mysecretpassword',
-
-      PG_WALLET_URL: 'localhost:5432',
-      PG_WALLET_MIN_IDLE_COUNT: '0',
-      PG_WALLET_MAX_CONNECTIONS: '90',
-      PG_WALLET_CONNECTION_TIMEOUT_MINS: '30'
-    }
-    const validationAsync = await util.promisify(validateAppConfig)
-    await validationAsync(appConfig)
+  it('should pass app config validation with AGENCY_TYPE enterprise', async () => {
+    const appConfig = getValidClientAgencyConfig()
+    await validateAppConfig(appConfig)
+  })
+
+  it('should pass app config validation with AGENCY_TYPE client', async () => {
+    const appConfig = getValidClientAgencyConfig()
+    await validateAppConfig(appConfig)
```

Comment:
These tests seem to do the same thing, the first one should use `getValidEnterpriseAgencyConfig `